### PR TITLE
docs(readme): updated as per new curriculum and certification

### PR DESCRIPTION
**Pull request by raisedadead**
Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/17361

### CODE REVIEW

**Comment by raisedadead** [comment]
@QuincyLarson Please take a look and edit the copy if your would like. I have taken this commit from @sanchit0496 's PR 

@sanchit0496 your's was from staging branch and got closed automatically when I rebased. But, don't worry you are still the author of the commit. I did not want you to lose the credit for this.

Next time I recommend always to work on a branch and never from the staging, to avoid such issues.

Thanks and happy contributing!

File: README.md [hunk 1]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
 
-Skills you'll practice include `HTML`, `CSS`, `JavaScript`, `jQuery` and `Bootstrap`.
 
-To earn this certification, you'll build **10 front-end projects** and implement many **JavaScript algorithms**.
+##### 1. Responsive Web Design
 
-##### 2. Data Visualization Certification
-The second section builds upon the first and introduces you to more advanced topics such as `Sass`, `React` and `D3`.
+**Add links to the live demos of your projects as you finish them. Then, once you have added all 5 projects required for a certification, you can claim it.**
+a. Build A Tribute Page
```
Comment (Bouncey) [inline README.md]: No need to label the challenges, we can just bullet-point them
```md
- item one
- item two
- item three
```
Becomes 
- item one
- item two
- item three

That is to say, they are all of equal importance

Comment (Bouncey) [inline README.md]: A link to each project challenge would be great :+1:

File: README.md [hunk 2]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+
+###These six new certifications will replace the old legacy certificates:
+
+**Legacy Front End Development**
+**Legacy Data Visualization**
+**Legacy Back End Development**
+
+###Coding for nonprofits
+
+We’ve decided to make coding for nonprofits a separate process from the certificate program. We’ve added 5 of our projects that are used by nonprofits - our Open Source for Good projects - to the challenge map. Feel free to browse their open issues. You can dive in and start contributing whenever you feel ready.
```
Comment (Bouncey) [inline README.md]: They are currently not in the challenge map, as far as I am aware.

File: README.md [hunk 3]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+
+We will continue to add Open Source for Good projects to the challenge map as they become stable and ready for your contributions. These projects are all built using the same JavaScript-based tools you’ll learn by earning the certificates. And several of these are nearing a public beta (I’ll formally announce these as they go into open beta).
+
+In addition to our Open Source for Good projects - which are particularly well-documented and beginner-friendly - there are tons of other open source projects you can contribute to. The community has built a guide to contributing to various open source projects here 363.
```
Comment (Bouncey) [inline README.md]: > The community has built a guide to contributing to various open source projects here 363.

This doesn't make sense? Is it lacking context?

File: README.md [hunk 4]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+
+###What happens to the old legacy certificates?
+
+If you’ve already earned a certificate, you will always have it. You will always be able to link to it from your LinkedIn or résumé. And when your prospective employers or freelance clients click that link, they’ll see a verified certificate specific to you.
+
+And if you’ve already started working toward the Legacy Front End Development, Legacy Data Visualization, or Legacy Back End Development certificates, and still want to claim it, don’t worry - you can will always be able to claim these certificates - even after we release the beta challenges.
```
Comment (Bouncey) [inline README.md]: There are no more `beta` challenges.

File: README.md [hunk 5]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+
+All of the required projects for these old certificates will remain available on freeCodeCamp. For the old certificates, you can complete old versions of the projects.
```
Comment (Bouncey) [inline README.md]: There are no 'old versions' of the projects. For example, the Tribute Page project is the Tribute Page project for both current and legacy certifications.

File: README.md [hunk 6]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+
+As long as you created your account before we release the expanded curriculum, you will be able to claim these old certificates any time you want, and they’ll start showing up on your freeCodeCamp profile.
```
Comment (Bouncey) [inline README.md]: There is currently no eligibility criteria for claiming the legacy certifications.

File: README.md [hunk 7]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+
+Campers who create their freeCodeCamp after we release the expanded curriculum will not be able to claim these old certificates.
```
Comment (Bouncey) [inline README.md]: The expanded curriculum has been released. Plus, see previous comment about eligibility.

File: README.md [hunk 8]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+
+Note that in order to claim the new certificates, you’ll need to build the new projects (some of which are updated versions of old projects) and get all of their tests passing.
```
Comment (Bouncey) [inline README.md]: We have no way of testing projects over on learn. It is for the camper to decide when the project meets the user stories/passes the built-in tests. Their project solutions will be publicly available if they want their certifications available publicly.

File: README.md [hunk 9]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+
+###Some cool facts about the new certificates
+
+1. Each of these certificates has 5 projects, all of which have built-in test suites. This means that each user story has a corresponding test that can tell you when you’ve properly fulfilled it.
+2. You can pull in these tests through a CDN, meaning you’ll be able to build these projects on websites like CodePen, or your local development environment, or even on freeCodeCamp.org 80 itself.
```
Comment (Bouncey) [inline README.md]: You can pull in these tests through a CDN, meaning you’ll be able to build these projects on websites like CodePen, or your local development environment ~, or even on freeCodeCamp.org 80 itself~.

This is not possible yet.

File: README.md [hunk 10]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+3. We love Node School but we’re getting rid of their challenges. We’ve built our own back end challenges and projects, which also have tests, and can be completed on Glitch.com 295 or your local development environment.
```
Comment (Bouncey) [inline README.md]: > and can be completed on Glitch.com 295 or your local development environment.

This doesn't make sense?

File: README.md [hunk 11]
```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
...
+4. We still recommend completing all of these certificates, but if you are already an experienced developer, it will be much easier for you to learn a specific skill you’re interested (like information security) without having to jump around through the map. All the topics will be broken out in as modular a way as possible.
```
Comment (Bouncey) [inline README.md]: We still recommend working your way through the challenges, but if you would like to learn a specific skill you’re interested in, feel free to skip ahead. All of the certifications can be completed in isolation, we do not lock off parts of the curriculum whilst earlier parts are incomplete.

File: README.md (hunk 10 above)
Comment (Bouncey) [inline README.md]: ~We love Node School but we’re getting rid of their challenges.~ We’ve built our own back end challenges and projects

**Review by Bouncey — CHANGES_REQUESTED** [review]
@sanchit0496 thank you for all of the effort you have put in to this PR, it is greatly appreciated.

As a general note `certificate -> certification` please.

**Comment by sanchit0496** [comment]
@Bouncey Ok, I'll make the necessary changes. Should I create a new branch to make the changes or do it on the staging branch itself?

**Comment by sanchit0496** [comment]
@raisedadead I'll be making the changes suggested above. Should I create a new branch for it, or do it on the staging branch itself?

**Comment by sanchit0496** [comment]
https://github.com/freeCodeCamp/freeCodeCamp/pull/17370
By mistake, I made a new pull request. I'm new to open source contribution and still learning that's why.
However, I've made the changes suggested by @Bouncey and made a new branch for making the changes instead of using the staging branch, as suggested by @raisedadead .

Refer:
https://github.com/freeCodeCamp/freeCodeCamp/pull/17367
https://github.com/freeCodeCamp/freeCodeCamp/issues/17361

**Comment by raisedadead** [comment]
Closing in favor of #17370

**Pull request closed without merging**